tests/tt_eager/profiling/reference.txt

```diff
@@ -256,7 +255,6 @@ ttnn.lt,40,0.9,1.19,1.24,1.12
 tt_lib.tensor.ltz,40,0.81,0.9,1.12,0.93
 tt_lib.tensor.mac,40,1.69,1.79,2.13,1.83
 tt_lib.tensor.maeloss,40,11.89,13.2,14.13,12.75
-tt_lib.tensor.matmul,40,0.88,1.78,3.89,1.5
```

Comment:
Are you sure you want to completely remove the reference for matmul?

Reply:
Yes.
This is for the python tt_lib operation that I am removing. I don't want any profiling tests trying to run a non-existent op.